cq-component-maven-plugin/src/main/java/com/citytechinc/cq/component/touchuidialog/widget/hidden/HiddenWidget.java

```diff
@@ -0,0 +1,38 @@
+/**
+ *    Copyright 2013 CITYTECH, Inc.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package com.citytechinc.cq.component.touchuidialog.widget.hidden;
+
+import com.citytechinc.cq.component.annotations.config.TouchUIWidget;
+import com.citytechinc.cq.component.annotations.widgets.Hidden;
+import com.citytechinc.cq.component.touchuidialog.widget.AbstractTouchUIWidget;
+import com.citytechinc.cq.component.touchuidialog.widget.DefaultTouchUIWidgetParameters;
+
+@TouchUIWidget(annotationClass = Hidden.class, makerClass = HiddenWidgetMaker.class,
+    resourceType = HiddenWidget.RESOURCE_TYPE)
```

Comment:
Since this resource type doesn't exist in 6.0 and 6.1 (at least not w/o service / feature packs) Can we add the `featureFlag` property here - maybe set it to something like `hiddenfieldwidget`.  See `com.citytechinc.cq.component.touchuidialog.widget.tagspicker.TagsPickerWidget` for an example of the usage of `featureFlag`.